addons/product/product.py

```diff
@@ -488,13 +505,12 @@ def _get_product_variant_count(self, cr, uid, ids, name, arg, context=None):
         'price': fields.function(_product_template_price, fnct_inv=_set_product_template_price, type='float', string='Price', digits_compute=dp.get_precision('Product Price')),
         'list_price': fields.float('Sale Price', digits_compute=dp.get_precision('Product Price'), help="Base price to compute the customer price. Sometimes called the catalog price."),
         'lst_price' : fields.related('list_price', type="float", string='Public Price', digits_compute=dp.get_precision('Product Price')),
-        'standard_price': fields.property(type = 'float', digits_compute=dp.get_precision('Product Price'), 
+        'standard_price': fields.function(_compute_product_template_field, fnct_inv=_set_product_template_field, type='float', string='Cost Price', digits_compute=dp.get_precision('Product Price'), 
```

Comment:
While you are at it, could'nt you make a "multi" field ?

Is it normal that it is not stored ?